src/test/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLORS = 7;
+
+    public static String coloring() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
static methods is bad practice
use verbs in Present Simple for method names